<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>397</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>RandomForest</string>
    </property>
    <widget class="QPlainTextEdit" name="address_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="plainText">
      <string>duetto.csv</string>
     </property>
    </widget>
    <widget class="QPushButton" name="train_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>440</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Train</string>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="kfold_text">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="plainText">
      <string>3</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="kfold_error">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>130</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="oob_error">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>190</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>190</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Out of bag Error</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Kfold Error</string>
     </property>
    </widget>
    <widget class="QLabel" name="kfold_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>K-Fold</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="number_of_trees">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="plainText">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of Trees</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Audio Files</string>
     </property>
    </widget>
    <widget class="QPushButton" name="open_button">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>280</y>
      <width>381</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Deep Learning</string>
    </property>
    <widget class="QPushButton" name="train_deeplearning_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Train</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hidden Layer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Epochs</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Learning Rate</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="hidden_layer_text">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="plainText">
      <string>2000</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="lr_text">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="plainText">
      <string>0.0001</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="epochs_text">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="plainText">
      <string>10</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="deep_learning_address_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="plainText">
      <string>mfcc.csv</string>
     </property>
    </widget>
    <widget class="QPushButton" name="open_button_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>381</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Extractor</string>
    </property>
    <widget class="QRadioButton" name="mfcc_radioButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>50</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>MFCC</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="filterbank_radioButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>FilterBank</string>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="wait_label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wait</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
